assets/js/atomic/blocks/product-elements/stock-indicator/supports.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import { isFeaturePluginBuild } from '@woocommerce/block-settings';
+
+export const supports = {
+	...( isFeaturePluginBuild() && {
+		color: {
+			text: true,
+			background: false,
+			link: false,
+		},
+	} ),
+	typography: {
+		fontSize: true,
```

Comment:
It makes sense to me to support all typography properties for this block instead of only font size. At least, font-weight and text-transform should be supported.

Reply:
According to main issue #4965, we should enable only font-size, but yeah we can enable them easily. @Aljullu, what do you think?

Reply:
Yes, makes sense to me. I tested the Post Categories block from Gutenberg and it offers many more typography options than what we offer in Stock indicator.

I guess the same could be said about the Product Category List, Product Summary and Product Tag List.

@vivialice what are your thoughts on this? Should we enable all typography properties in those blocks?